google/api_core/protobuf_helpers.py

```diff
@@ -63,8 +63,8 @@ def from_any_pb(pb_type, any_pb):
     # Unpack the Any object and populate the protobuf message instance.
     if not any_pb.Unpack(msg_pb):
         raise TypeError(
-            "Could not convert {} to {}".format(
-                any_pb.__class__.__name__, pb_type.__name__
+            "Could not convert\n   Any[{}] to\n       {}".format(
```

Comment:
`Any` is designed to represent any type without constraints so this syntax doesn't seem appropriate.

Reply:
As a follow up, can you add a unit test for this change so it's easier to verify the generated error?

Reply:
Makes sense. Thanks guys.